crates/pallet-domains/src/lib.rs

```diff
@@ -1443,6 +1443,28 @@ impl<T: Config> Pallet<T> {
             }
         });
     }
+
+    /// Returns if there are any ERs in the challenge period that have non empty extrinsics.
+    pub fn non_empty_extrinsic_receipts_exists(domain_id: DomainId) -> bool {
```

Comment:
We are not actually checking ER here perhaps `non_empty_domain_block_exists` or `non_empty_bundle_exists` would be more appropriate.